tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -961,13 +964,23 @@ def writeset(self, max_size):
         # Fill up the initial nodes with data, checkpoint them and stop
         # them. Then bring them back up and ensure the checkpoint data is
         # there.
-        client1 = self.bft_network.random_client()
+
         # Write enough data to checkpoint and create a need for state transfer
-        for i in range(1 + checkpoints_num * 150):
-            key = self.skvbc.random_key()
-            val = self.skvbc.random_value()
-            kv = [(key, val)]
-            await self.write_and_track_known_kv(kv, client1)
+        max_concurrency = len(self.bft_network.clients) // 2
+        clients = self.bft_network.random_clients(max_concurrency)
+        iterations = checkpoints_num * 150
+        replica_id = 0
+        last_executed_seq_num = 0
+        keys = ['replica', 'Gauges', 'lastExecutedSeqNum']
+
+        while last_executed_seq_num < iterations:
```

Comment:
Let's drop the `iterations` variable and have `checkpoints_num * 150` here directly.